### PR TITLE
Add boolean on partner to identify consignees and create views related t...

### PR DESCRIPTION
**Pull request by yvaucher**
...o this change

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

File: logistic_consignee/view/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data noupdate="0">
+
+    <record id="base_ifrc_partner_form_view_consignee_check_box" model="ir.ui.view">
+      <field name="name">ifrc base partner view consignee check box</field>
+      <field name="model">res.partner</field>
+      <field name="inherit_id" ref="base.view_partner_form"/>
```
Comment (gurneyalex) [inline logistic_consignee/view/res_partner.xml:8]: it's cleaner to mention a dependency on base in that case.

**Comment by gurneyalex** [comment]
globally :+1:

File: logistic_consignee/view/res_partner.xml (hunk 1 above)
Comment (nbessi) [inline logistic_consignee/view/res_partner.xml:8]: agreed

**Comment by nbessi** [comment]
:+1:  It will be nice to fix @gurneyalex  comment

**Comment by yvaucher** [comment]
@nbessi @lepistone fixed

**Comment by lepistone** [comment]
:+1: . I'll merge once green.

**Pull request merged**